Dislike.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+require_once 'SessionHandler.php';
+require_once 'src/Models/PostModel.php';
+require_once 'connectToDB.php';
+
+session_start();
+
+$session = new SessionHandles();
+if (!$session->checkUserLoggedIn()) {
+    header('Location: login.php');
+} else {
+    $postId = $_GET['id'];
+    header("Location: singlePost.php?id={$_GET['id']}");
```

Comment:
You should move this header to the end of the else - always good to have it be the last thing just to be safe